stargate/Dockerfile

```diff
@@ -23,9 +22,9 @@ FROM alpine:latest
 # TODO: Unsure if needed
 RUN apk --no-cache add ca-certificates
 COPY --from=builder \
-  /home/rust/src/target/x86_64-unknown-linux-musl/release/stargate \
+  /home/rust/src/target/x86_64-unknown-linux-musl/release/apollo-stargate \
   /usr/local/bin/
 
 # User can provide an alternate port via "-e 'PORT=<your port>'" when running the container.
 # PORT defaults to 8080 otherwise.
-CMD /usr/local/bin/stargate --address=http://0.0.0.0:${PORT:-8080} --manifest=/graph
+CMD /usr/local/bin/apollo-stargate --port=${PORT:-8080} --manifest=${CSDL:-"/graph.graphql"}
```

Comment:
Well, so this is an interesting bit. Are you asking that we change this so that the user facing binary we release is just `stargate`?
This is getting the name from the crate name, and at some point we said that we want to prefix those with `apollo-`.

If we release Stargate as a binary, we can rename the file to whatever we want as part of the download / extraction code.

I think that we might want to, at some point, publish the binary to crates.io, at which point the crate name will be the "user facing" name, and rust folks could install it with `cargo install stargate` or `cargo install apollo-stargate` depending on what we end up with.

I see that `stargate` is available, so maybe we should actually go ahead and squat on that name..

Gonna change it!